src/coreclr/src/inc/readytorun.h

```diff
@@ -51,29 +45,35 @@ enum ReadyToRunFlag
     READYTORUN_FLAG_NONSHARED_PINVOKE_STUBS     = 0x00000008    // PInvoke stubs compiled into image are non-shareable (no secret parameter)
 };
 
-enum ReadyToRunSectionType
+enum class ReadyToRunSectionType : DWORD
```

Comment:
Nit: It would be better to use C/C++ types like `uint32_t` instead of the Windowsism.

Reply:
Fixed in 2nd commit, thanks for pointing that out.